rules-tests/Restoration/Rector/ClassLike/UpdateFileNameByClassNameFileSystemRectorUsingMultipleFilesChangedTrait/config/configured_rule.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Restoration\Rector\ClassLike\UpdateFileNameByClassNameFileSystemRector;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+    $services->set(UpdateFileNameByClassNameFileSystemRector::class);
```

Comment:
I have a feeling I have troubles to express myself clearly. I'm sorry if that's the case.

I'll try an example. Let me me know if it can be improved :+1: 

The test must be independend on any existing rule. That means it should be in a new directory, with 0 % shared code. In a file structure, it should look like this:

```bash
/tests
    /PHPUnit/
        /MultipleFilesChangedTrait
             /config
                  configured_rule.php
             /Rector
                  RenameFileToWhateverRector.php
             /Fixture
                  some_fixture.php.inc
             MultipleFilesChangedTraitTest.php
```

Reply:
OK, I've created test in suggested structure now.

The reason why I didn't do it before is what you wrote me several months ago here https://github.com/rectorphp/rector/pull/5520#issuecomment-778402362

> We'll need a test fixture without creating a new Rector rule. Then we'd fix the custom rule, and not a Rector bug.
> Please re-use existing one with auto import enabled.

I thought that it is some rule that we should not create Rector rules just for tests.